internal/pkg/cli/svc_deploy.go

```diff
@@ -696,21 +709,33 @@ func (o *deploySvcOpts) deploySvc(addonsURL string) error {
 		return err
 	}
 
+	if o.cmdRunAt == nil {
+		o.cmdRunAt = aws.Time(time.Now())
+	}
```

Comment:
nit: another way I think of achieving this would be

```go
type deploySvcOpts struct {
   ...
   now   func() time.Time
}

func newSvcDeployOpts(vars deployWkldVars) (*deploySvcOpts, error) {
   opts.now = time.Now
}
```
and in the tests we can put a `fakeNow` function